README.md

```diff
@@ -1,6 +1,52 @@
 # @ember-decorators/auto-computed
 
-This README outlines the details of collaborating on this Ember addon.
+Note: If you are using getter and setter functions (the recommended way), please use the `@computed` key from [ember-decorators](https://github.com/ember-decorators/ember-decorators)
+
+### Installation
+
+`ember install @ember-decorators/auto-computed`
+
+If you're using **@ember-decorators/auto-computed** in an addon, pass the `-S` option to save
+this as a `dependency` as opposed to the default `devDependency`:
+
+`ember install -S @ember-decorators/auto-computed`
+
+### Application Usage
+
+In your application where you would normally have:
+
+```javascript
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  bar: Ember.computed('someKey', 'otherKey', function() {
+    var someKey = this.get('someKey');
+    var otherKey = this.get('otherKey');
+
+    return `${someKey} - ${otherKey}`;
+  })
+})
+
+```
+
+You replace it with this:
+
+```javascript
+import Component from '@ember/component';
+import autoComputed from 'auto-computed';
```

Comment:
Put this here for documentation.  Was thinking that if a user was using both `@computed` and this, it might make it easier to find in the future.  Thoughts?

Reply:
Do you mean the documentation itself? Or the name `@autoComputed` vs `@computed`? I agree that `@autoComputed` is better, it'll be easier to find and to know the difference between the two.

One thing to note, we should follow the package naming conventions from other ember-decorators projects, so the imports should come from the `@ember-decorators` namespace, e.g.

```js
import autoComputed from '@ember-decorators/auto-computed';
```

You can do this by changing the name of the package in `package.json` and `index.js`. We also have been using named exports instead of `default` for most things that aren't classes, [following the convention established by the Ember JS modules RFC](https://github.com/emberjs/rfcs/blob/master/text/0176-javascript-module-api.md#classes-are-default-exports), but I'm open here since `autoComputed` is definitely the main point of this module.

Reply:
Cool.  Yeah open to either way.  You let me know which way you want it - default or named!